examples/demo_mult_image_sockets2.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+# title           : this_python_file.py
+# description     :This will create a header for a python script.
+# author          :bryant.chhun
+# date            :12/19/18
+# version         :0.0
+# usage           :python this_python_file.py -flags
+# notes           :
+# python_version  :3.6
+
+
+import sys
+from PyQt5.QtCore import pyqtSignal, pyqtSlot, QObject, QRunnable, QThreadPool
+from PyQt5.QtWidgets import QApplication, QWidget
+from napari_gui import Window, Viewer
+
+import numpy as np
+import time
+
+"""
```

Comment:
docstring should go at the top of the file

Reply:
removing this for now